### DISCUSSION TITLE
pyinstaller for ARM64 aarch system failes to install/open.

### DISCUSSION THREAD

**Discussion by MelkonTech** [pyinstaller/pyinstaller#7401] [Help]
Let's say there is a simple pyQt5 app which is running fine on the AMD64 machine.
When using a version which was built into the AMD64 system, it gives an error that AMD64 is not supported.
Using a bootloader (in a way it is mentioned [here](https://pyinstaller.org/en/stable/bootloader-building.html)) or building the app inside an ARM64 system also doesn't help - as the app installs without an error but when you try to open it, it tries to open a few seconds and then closes.
Does anyone have any idea about the issue and how to build a working compiled pyqt app using pyinstaller? 

Some references related to the errors.
https://github.com/pyinstaller/pyinstaller/issues/2848
https://unix.stackexchange.com/questions/420008/installing-pyqt5-on-linux-failed-due-to-sip-dependency
https://forum.qt.io/topic/118699/pip-install-pyqt5-gives-tons-of-errors-why/5
https://github.com/altendky/pyqt-tools/issues/100

**Comment by bwoodsend** [comment]
To build an arm64 application, you need to run PyInstaller on an arm64 machine.
